### ISSUE TITLE
Add ", or" to first Exception Understanding 2.4.11: Focus Appearance (Minimum)

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2459]
Add ", or" to first Exception Understanding 2.4.11: Focus Appearance (Minimum)
https://w3c.github.io/wcag/understanding/focus-appearance-minimum.html#exceptions

Just as is done in the SC text itself, an ", or" should be aaded to Understanding doc:

Change:

> the focus indicator cannot be adjusted by the author,
> the author has not modified the effects of the user agent default, or
> the author provides an indicator of sufficient size and contrast.

To:

> the focus indicator cannot be adjusted by the author, **or**
> the author has not modified the effects of the user agent default, or
> the author provides an indicator of sufficient size and contrast.

**Comment by alastc** [comment]
The [understanding doc](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html) is updated.
